### DISCUSSION TITLE
How does Frigate decide which stream to use for live view?

### DISCUSSION THREAD

**Discussion by matt-garman** [blakeblackshear/frigate#14463] [General]
I'm running two instances of Frigate 14.1, with very similar configs.  Both appear to be working as-expected.

However, when using live view for any of my cameras, one Frigate instance show the high resolution/high FPS stream (i.e. the one I have set for recording).  For the other instance, when I do live view of a camera, it is showing the low resolution/low FPS stream (i.e. the one that is for detection).

My question is: how does Frigate decide which stream to show in live view mode?  I seem to recall in version 13, you could explicitly select which stream you wanted to see in live view mode.

My guess is that it's somehow dynamically determined in v14.  There is a big performance difference in the hardware running my two instances, and the instance showing the low-res stream is the weaker system.  If indeed this is dynamically determined, is there a way to force the high resolution stream?

**Comment by hawkeye217** [comment]
See the Live view docs: https://docs.frigate.video/configuration/live/

Beyond what the docs say, we'd have to see your config to explain exactly what's happening in your case.

**Reply by matt-garman** [reply]
Thanks!  I changed my second instance (the one displaying the low(er) resolution stream in live view) to include a go2rtc config similar to the first instance, and now it is indeed displaying the full resolution stream in live view.

My issue was that I wrongly assumed go2rtc was unnecessary in the second instance, since it was originally pulling video from the first instance's go2rtc (rather than from the cameras directly).

Now the second instance is still getting video from the first instance's go2rtc, but additionally has its own go2rtc instances.

Thanks again!

**Reply by hawkeye217** [reply]
Great! Glad it's working.
